### DISCUSSION TITLE
Add option to turn infobox tooltips off/add delay

### DISCUSSION THREAD

**Discussion by Tomato3s** [runelite/runelite#15063] [Ideas]
Instead of the tooltips appearing as soon as you hover your mouse over an infobox like stamina potion timer, add a delay before the tooltip shows up. 1 second or so? Or add an option to turn them off.  It's pretty annoying when you're doing something where you constantly need to move your mouse past them.

**Comment by alexanderhenne** [comment]
Or maybe a key that you must hold for the tooltip to appear. (toggleable)

**Comment by deathbeam** [comment]
Probably configurable delay would be best option, and make the default 0
